providers/edu/harvarddataverse/normalizer.py

```diff
@@ -9,33 +9,21 @@ class Person(Parser):
 
 
 class Contributor(Parser):
```

Comment:
For authors, this should be `Creator`, and should have `cited_as` and `order_cited` fields.